<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Stock Manager</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>731</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_current">
    <attribute name="title">
     <string>Current</string>
    </attribute>
    <widget class="QTableWidget" name="currentTableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>26</y>
       <width>721</width>
       <height>441</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Item</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Buy Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Current Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Profit($)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Profit(%)</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </brush>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="5">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="6">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="currentInsertBtn">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>2</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="currentDelBtn">
     <property name="geometry">
      <rect>
       <x>689</x>
       <y>2</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="currentSaveBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>527</x>
       <y>-1</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="currentCancelBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>578</x>
       <y>-1</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_history">
    <attribute name="title">
     <string>History</string>
    </attribute>
    <widget class="QTableWidget" name="historyTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>26</y>
       <width>721</width>
       <height>451</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Buy Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sell Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sell Qunatity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Profit($)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Profit(%)</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="historyDelBtn">
     <property name="geometry">
      <rect>
       <x>689</x>
       <y>2</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="historyInsertBtn">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>2</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="historySaveBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>527</x>
       <y>-1</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="historyCancelBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>578</x>
       <y>-1</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="updateTime">
   <property name="geometry">
    <rect>
     <x>748</x>
     <y>14</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>updated 2018-01-02 12:02:23</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>33</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Investment Period</string>
   </property>
   <widget class="QLabel" name="investPeriod">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>23</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>123 days</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QTreeWidget" name="portfolioTreeWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="itemsExpandable">
    <bool>true</bool>
   </property>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>10</number>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Portfolio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Total Input</string>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>230</red>
       <green>230</green>
       <blue>230</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Current Value</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
    </property>
    <item>
     <property name="text">
      <string>Stock</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cash</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>230</red>
        <green>230</green>
        <blue>230</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Profit($)</string>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Realized Profit($)</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>245</red>
       <green>255</green>
       <blue>171</blue>
      </color>
     </brush>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>245</red>
       <green>255</green>
       <blue>171</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Realized Profit(%)</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>245</red>
       <green>255</green>
       <blue>171</blue>
      </color>
     </brush>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>245</red>
       <green>255</green>
       <blue>171</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>updateBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>update_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_data()</slot>
 </slots>
</ui>
